src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,295 @@ metadata_model(
     path = "$PKG.LinearCountRegressor"
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+[`LinearRegressor`](https://en.wikipedia.org/wiki/Linear_regression) assumes the target is a continuous variable
```

Comment:
I suggest we drop the wiki entry for linear regressor as it might possibly lead to confusion about the specific algorithm being used. We already have a reference to the GLM package in the auto-generated header, which I expect is sufficient.

Reply:
Perfect!